lib/promscrape/targetstatus.qtpl

```diff
@@ -96,9 +96,10 @@
 {% endfunc %}
 
 {% func commonHeader() %}
+    {% code pathPrefix := httpserver.GetPathPrefix() %}
```

Comment:
this is only a part of the solution. You also need to account for any link used in vmagent's UI:
```
vmagent landing page does not modify the presented links with the -http.pathPrefix option
```